user/useradm.go

```diff
@@ -89,6 +89,8 @@ type App interface {
 type Config struct {
 	// token issuer
 	Issuer string
+	// white-listed issuers
+	AllowedIssuers map[string]bool
```

Comment:
perf: To declare a hashmap for a set, you can use the empty struct:
```suggestion
	AllowedIssuers map[string]struct{}
```

Reply:
okey.